### PR TITLE
feat(2267): Trim unused logic

### PR DESCRIPTION
**Pull request by klu909**
## Context

Currently a large chunk of logic in collection endpoint are add useless information to api response that was not being used anywhere in ui

## Objective

Trim useless logics

## References

https://github.com/screwdriver-cd/screwdriver/issues/2267

## License
I confirm that this contribution is made under a BSD license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by DekusDenial — APPROVED** [review]
i like the red in this

File: plugins/collections/get.js [hunk 1]
```diff
@@ -168,12 +51,12 @@ module.exports = () => ({
 
                     return (
                         Promise.all(collectionPipelines)
-                            // Populate pipelines with PR Info and then last builds
-                            .then(pipelines => Promise.all(getPipelinesInfo(pipelines, eventFactory)))
-                            .then(pipelinesWithInfo => {
-                                result.pipelines = pipelinesWithInfo;
+                            // to filter out null
+                            .then(pipelines => pipelines.filter(pipeline => pipeline))
```
Comment (adong) [inline plugins/collections/get.js:55]: What does filter do here?

`pipelines.filter(pipeline => pipeline))`

 Is it to filter out nullish values?

**Review by jithine — APPROVED** [review]

File: plugins/collections/get.js [hunk 2]
```diff
@@ -5,123 +5,6 @@ const joi = require('joi');
...
-function getPipelinesInfo(pipelines, eventFactory) {
-    return (
-        pipelines
-            // Filter out pipelines that don't exist
-            .filter(pipeline => !!pipeline)
-            .map(pipeline =>
-                // Get the PR Info and last builds for each pipeline
-                Promise.all([getPipelinePRInfo(pipeline), getPipelineHealth(pipeline, eventFactory)])
-                    // Combine the PR Info and last builds and return the populated pipeline
-                    .then(([pipelinePRInfo, pipelineHealth]) => {
```
Comment (tkyi) [inline plugins/collections/get.js:114]: are we not planning to show any of this data in the collections?

Comment (jithine) [reply plugins/collections/get.js:114]: They are not used anywhere, this was was before we had build metrics. for the old collections view.

File: plugins/collections/get.js (hunk 1 above)
Comment (klu909) [reply plugins/collections/get.js:55]: yes

**Pull request merged**